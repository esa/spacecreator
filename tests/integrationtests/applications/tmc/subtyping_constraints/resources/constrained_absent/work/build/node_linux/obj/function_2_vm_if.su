function_2_vm_if.c:14:6:init_function_2	32	static
function_2_vm_if.c:26:6:function_2_PI_1	48	static
function_2_vm_if.c:57:1:_GLOBAL__sub_D_00099_0_function_2_initialized	16	static
function_2_vm_if.c:57:1:_GLOBAL__sub_I_00099_1_function_2_initialized	16	static
