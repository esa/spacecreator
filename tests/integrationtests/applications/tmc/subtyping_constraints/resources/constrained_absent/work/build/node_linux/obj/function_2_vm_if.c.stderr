[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/function_2/SDL/wrappers/function_2_vm_if.c:[m[K In function â€˜[01m[Kinit_function_2[m[Kâ€™:
[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/function_2/SDL/wrappers/function_2_vm_if.c:18:17:[m[K [01;35m[Kwarning: [m[Kunused variable â€˜[01m[Kunused[m[Kâ€™ [[01;35m[K-Wunused-variable[m[K]
       long long [01;35m[Kunused[m[K = getTimeInMilliseconds();
                 [01;35m[K^~~~~~[m[K
[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/function_2/SDL/wrappers/function_2_vm_if.c:[m[K In function â€˜[01m[Kfunction_2_PI_1[m[Kâ€™:
[01m[K/home/taste/subtyping-projects-final/constrained_absent/work/function_2/SDL/wrappers/function_2_vm_if.c:27:34:[m[K [01;35m[Kwarning: [m[Kunused parameter â€˜[01m[KIN_p1_len[m[Kâ€™ [[01;35m[K-Wunused-parameter[m[K]
       (const char *IN_p1, [01;35m[Ksize_t IN_p1_len[m[K)
                           [01;35m[K~~~~~~~^~~~~~~~~[m[K
